recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb

```diff
@@ -13,11 +11,11 @@ FILES_${PN} += "${base_prefix}/${IMAGES_DST_DIR}"
 # force install so if DomU kernel changes we use the latest binaries
 do_install[nostamp] = "1"
 do_install() {
-   install -d "${D}/${base_prefix}/${IMAGES_DST_DIR}"
-   find "${XT_SHARED_ROOTFS_DIR}/${IMAGES_SRC_DIR}" -iname 'Image*' -exec \
-     cp -f --no-dereference --preserve=links {} "${D}/${base_prefix}/${IMAGES_DST_DIR}" \;
+   install -d "${D}/${base_prefix}/${XT_DIR_ABS_ROOTFS_DOMD}"
```

Comment:
You removed `/` at L5, but left it there.

Reply:
done